### PR TITLE
Exposed channel target and call peer in PHP

### LINKED ISSUE #2545: Expose call peer address in PHP
#2542 exposes this data from C, add some wrapping API for client and server

### PR DESCRIPTION
**Pull request by stanley-cheung**
Fixes #2545

### CODE REVIEW

File: src/php/tests/interop/interop_client.php [hunk 1]
```diff
@@ -332,10 +332,11 @@ function timeoutOnSleepingServer($stub) {
   $opts['update_metadata'] = $auth->getUpdateMetadataFunc();
 }
 
-$stub = new grpc\testing\TestServiceClient(
-    new Grpc\BaseStub(
-        $server_address,
-        $opts));
+$internal_stub = new Grpc\BaseStub($server_address, $opts);
+hardAssert($internal_stub->getTarget() == $server_address,
```
Comment (murgatroid99) [inline src/php/tests/interop/interop_client.php]: This isn't guaranteed. As far as I understand, we only know that it is functionally equivalent.

File: src/php/tests/unit_tests/CallTest.php [hunk 2]
```diff
@@ -79,4 +79,8 @@ public function testAddSingleAndMultiValueMetadata() {
     $result = $this->call->startBatch($batch);
     $this->assertTrue($result->send_metadata);
   }
+
+  public function testGetPeer() {
+    $this->assertTrue($this->call->getPeer() == 'localhost:' . self::$port);
```
Comment (murgatroid99) [inline src/php/tests/unit_tests/CallTest.php]: see above.

File: src/php/tests/unit_tests/EndToEndTest.php [hunk 3]
```diff
@@ -149,4 +149,8 @@ public function testClientServerFullRequestResponse() {
     unset($call);
     unset($server_call);
   }
+
+  public function testGetTarget() {
+    $this->assertTrue($this->channel->getTarget() == 'localhost:' . $this->port);
```
Comment (murgatroid99) [inline src/php/tests/unit_tests/EndToEndTest.php]: see above.

File: src/php/tests/interop/interop_client.php (hunk 1 above)
Comment (stanley-cheung) [inline src/php/tests/interop/interop_client.php]: Got it. Fixed. Changed all those to just `is_string` checks. Thanks for the feedback.

**Pull request merged**